webapp/package-lock.json

```diff
@@ -1561,15 +1561,6 @@
         "eslint": ">=4.19.1"
       }
     },
-    "boards/node_modules/eslint-plugin-eslint-comments/node_modules/escape-string-regexp": {
```

Comment:
coud you please revert these changes

Reply:
Yes, no problem.

Reply:
I'm  running into an issue.  These updates happened because I ran npm install when I cloned the project, but if I don't do that then the web-server doesn't run because it "cannot find module 'chalk'", I though of running npm ci however this will delete the node_modules.  Do you have any recommendations? Thank you.

Reply:
yes you can revert these manually by copying the files from `master` branch